app/controllers/outgoing_messages/delivery_statuses_controller.rb

```diff
@@ -0,0 +1,30 @@
+# -*- encoding : utf-8 -*-
+class OutgoingMessages::DeliveryStatusesController < ApplicationController
+  before_filter :set_outgoing_message, :check_prominence
+
+  def show
+    @title = _('Delivery Status for Outgoing Message #{{id}}',
```

Comment:
Yeah it was more that I couldn't think of anything better to get the message across that the status is for the individual message rather than the requests itself. The ID is exposed in the URL so I don't think there's a problem in that respect. Going to just go with this for now.